lib/tasks/pkg.rake

```diff
@@ -24,5 +24,20 @@ namespace :pkg do
     version = `git show #{ref}:VERSION`.chomp.chomp('-develop')
     raise "can't find VERSION from #{ref}" if version.length == 0
     `git archive --prefix=foreman-#{version}/ #{ref} | bzip2 -9 > pkg/foreman-#{version}.tar.bz2`
+
+    `npm install`
+    `node_modules/.bin/webpack --bail --config config/webpack.config.js`
+    `mv node_modules pkg/`
+    `cp -rf public/webpack pkg/`
+
+    Dir.chdir('pkg') do
+      `bunzip2 *.tar.bz2`
```

Comment:
I think instead of bzipping, unzipping and bzipping again we could remove the first bzip and just add the files to the tarball before bzipping it.